src/daemon/abrt-action-save-package-data.c

```diff
@@ -320,7 +323,8 @@ static int SavePackageDescriptionToDebugDump(const char *dump_dir_name, const ch
     /* if basename is known interpreter, we want to blame the running script
      * not the interpreter
      */
-    if (g_list_find_custom(settings_Interpreters, basename, (GCompareFunc)g_strcmp0))
+    if (g_regex_match_simple(DEFAULT_INTERPRETERS_REGEX, basename, G_REGEX_EXTENDED, /*MatchFlags*/0) ||
+        g_list_find_custom(settings_Interpreters, basename, (GCompareFunc)g_strcmp0))
```

Comment:
Is the cast for `g_strcmp0()` really needed? I forget my C-fu, but the signatures should be compatible here.

Reply:
```
abrt-action-save-package-data.c: In function 'SavePackageDescriptionToDebugDump':
abrt-action-save-package-data.c:327:61: error: passing argument 3 of 'g_list_find_custom' from incompatible pointer type [-Werror=incompatible-pointer-types]
  327 |         g_list_find_custom(settings_Interpreters, basename, g_strcmp0))
      |                                                             ^~~~~~~~~                   
      |                                                             |                                
      |                                                             int (*)(const char *, const char *)
In file included from /usr/include/glib-2.0/glib/ghash.h:33,
                 from /usr/include/glib-2.0/glib.h:50,                                                                 
                 from abrt-action-save-package-data.c:20:
/usr/include/glib-2.0/glib/glist.h:122:25: note: expected 'GCompareFunc' {aka 'int (*)(const void *, const void *)'} but argument is of type 'int (*)(const char *, const char *)'
  122 |       GCompareFunc      func);
      |       ~~~~~~~~~~~~~~~~~~^~~~                                                                                   
cc1: all warnings being treated as errors
```